### ISSUE TITLE
Running but not streaming/injecting

### ISSUE THREAD

**Issue by filipemerker** [BrowserSync/browser-sync#683]
I'm experiencing a big trouble here. I Already had look up every issue around and everyonde gets it working but not me. That's my code

```javascript
'use strict';

var gulp        = require('gulp'),
    uglify      = require('gulp-uglify'),
    minify      = require('gulp-minify-css'),
    concat      = require('gulp-concat'),
    sass        = require('gulp-sass'),
    browserSync = require('browser-sync').create(),
    plumber     = require('gulp-plumber'),
    spritesmith = require('gulp.spritesmith'),
    neat        = require('node-neat').includePaths;

//////////////////////////////
// PATHS
//////////////////////////////
var path = {
  sassWatch: [
    'assets/scss/*.scss',
    'assets/scss/**/*.scss'
  ],
  sass_src: 'assets/scss/*.scss',
  sass_dest: 'dist/css',
  js_lint_src: [
      'dist/js/*.js'
  ],
  js_src : [
      'assets/js/*.js'
  ],
  js_dest : 'dist/js/'
};

//////////////////////////////
// BrowserSync
//////////////////////////////
gulp.task('browser-sync', function() {
  //watch files
    browserSync.init(null, {
        files: ['dist/**/*.*'],
        proxy: "http://mmda-treinamento.l"
    });
});

gulp.task('bs-reload', function (){
    browserSync.reload();
});

//////////////////////////////
// JS Tasks
//////////////////////////////
gulp.task('js', function () {
    gulp.src(path.js_src)
        .pipe(uglify())
        .pipe(concat('main.js'))
        .pipe(gulp.dest('dist/js'));
});

//////////////////////////////
// SASS Tasks
//////////////////////////////
gulp.task('sass', function(){
    gulp.src(path.sass_src)
        .pipe(sass({
            includePaths: ['styles'].concat(neat)
        }))
        .pipe(minify())
        .pipe(concat('style.css'))
        .pipe(gulp.dest('dist/css'))
        .pipe(browserSync.reload({stream: true}))
});

//////////////////////////////
// SPRITE Tasks
//////////////////////////////
gulp.task('sprite', function () {
  var spriteData = gulp.src('dist/img/sprite/*.png')
  .pipe(spritesmith({
    imgName: 'sprite.png',
    cssName: 'sprite.scss',
    optimizationLevel: 7
  }));
  spriteData.img.pipe(gulp.dest('dist/img/'));
  spriteData.css.pipe(gulp.dest('assets/scss/config/'));
});

//////////////////////////////
// Watch Tasks
//////////////////////////////
gulp.task('watch', ['browser-sync'], function () {
    gulp.watch(path.js_src, ['js']);
    gulp.watch(path.sassWatch, ['sass']);
    gulp.watch('gulpfile.js', ['default']);
    gulp.watch('assets/img/sprite/*.*', ['sprite']);
});

//////////////////////////////
//Default Tasks
//////////////////////////////
gulp.task('default', [
    'sass',
    'js',
    'watch',
    'sprite'
]);
```
And terminal returns me that
```Shell
[13:51:40] Using gulpfile /www/mmda-treinamento/sites/all/themes/teste/gulpfile.js
[13:51:40] Starting 'sass'...
[13:51:40] Finished 'sass' after 9.31 ms
[13:51:40] Starting 'js'...
[13:51:40] Finished 'js' after 3.61 ms
[13:51:40] Starting 'browser-sync'...
[13:51:40] Finished 'browser-sync' after 9.46 ms
[13:51:40] Starting 'watch'...
[13:51:40] Finished 'watch' after 15 ms
[13:51:40] Starting 'sprite'...
[13:51:40] Finished 'sprite' after 2.15 ms
[13:51:40] Starting 'default'...
[13:51:40] Finished 'default' after 6.13 μs
[BS] 1 file changed (style.css)
[BS] 1 file changed (style.css)
[BS] Proxying: http://mmda-treinamento.l
[BS] Access URLs:
 ----------------------------------
       Local: http://localhost:4006
    External: http://10.0.0.29:4006
 ----------------------------------
          UI: http://localhost:3058
 UI External: http://10.0.0.29:3058
 ----------------------------------
[BS] Watching files...

```
And every scss file that i edit, it returns
```Shell
[13:51:58] Starting 'sass'...
[13:51:58] Finished 'sass' after 3.63 ms
[BS] 1 file changed (style.css)
[BS] File changed: dist/css/style.css
```
But it neve got any close of injecting something in the page.
The BrowserSync itself works, if i force the reload, it work, but never injection.

I've lost my mind already, after two days of tests. Please someone help me.
Tnx

Also, my folder strucutre is:
theme
theme/assets (dev files)
theme/assets/js
theme/assets/scss
theme/assets/scss/base (and config and whatever)
theme/dist (production files)
theme/dist/css
theme/dist/js

**Comment by shakyShane** [comment]
1. Do you see 'Connected to Browsersync' message every time you reload the browser (top right)?
2. Does your page contain a `link` element with `style.css` as the `href`?

**Comment by filipemerker** [comment]
1. Yes.
2. No, it has a <style>@import </style>

**Comment by shakyShane** [comment]
Ah ok, please see https://github.com/BrowserSync/browser-sync/issues/10

tl;dr - we do not currently support this, but we will do soon.

**Comment by ryentzer** [comment]
I'm importing with @imports  into style.css from assets/css/main.css. However, the page won't reload. Is this the same issue? It's caused days worth of frustration. Just came across this.

**Comment by ArtemVPankov** [comment]
Hi! insert this <script async="" src="/browser-sync/browser-sync-client.js"></script>   before </body> if u use proxy on browserSync function
